src/ApiServices/ApiServices.js

```diff
@@ -62,4 +62,163 @@ const getNotedata = async () => {
   }
 };
 
-export { signUpHandler, logInHandler, createNoteHandler, getNotedata };
+// Archive
+// Add to archive
+const addToArchiveHandler = async (note) => {
+  try {
+    const res = await axios.post(
+      `/api/notes/archives/${note._id}`,
+      {
+        note: { ...note },
+      },
```

Comment:
After ES6 passing just `note` will do the same thing.
For Eg. - `{ note: {...note}}` can be written as `{note}`